### DISCUSSION TITLE
How to change the default saving directory?

### DISCUSSION THREAD

**Discussion by mastertap3** [pajoma/vscode-journal#74] [Q&A]
Hi,

I am not a coder, just a n00b using vscode for betterfountain and I thought this journal was a cool idea. Thank you for this.

Can you just help me with figuring out how I can save the journal folder in my E: drive rather in the default C: ?

**Comment by pajoma** [comment]
You can set the base path in the settings. The default base path is the folder "Journal" in your user profile directory. To change the base path, follow these steps

1. Go to File -> Settings (or `Ctrl+'y'`)
2. Search for "journal"
3. Edit the Base property (just copy the path from Windows Explorer)
